os/drivers/audio/syu645bscripts.h

```diff
@@ -45,7 +45,7 @@ typedef struct {
 
 t_codec_init_script_entry codec_reset_script[] = {
 	/* 0x0F for soft reset */
-	{ 0x0F, {0x01,}, 0, SYU645B_REG_D_2BYTE}
+	{ 0x0F, {0x01,}, 30, SYU645B_REG_D_2BYTE}
```

Comment:
what is this 30? Is this related to delay?

Reply:
yes , it adds a delay of that millisecond

Reply:
which page it mentioned?